src/libbson/src/bson/bson-atomic.h

```diff
@@ -372,31 +388,31 @@ DECL_ATOMIC_INTEGRAL (int64, __int64, 64)
 DECL_ATOMIC_STDINT (int64, 64)
 #endif
 #else
-static BSON_INLINE int64_t
+static BSON_INLINE int64_t BSON_GNUC_DEPRECATED
 bson_atomic_int64_fetch (const int64_t volatile *val, enum bson_memory_order order)
```

Comment:
```suggestion
static BSON_INLINE int64_t
bson_atomic_int64_fetch (const int64_t volatile *val, enum bson_memory_order order) BSON_GNUC_DEPRECATED
```
For consistency? (Deprecation macro at end of function declaration.)